### PR TITLE
Updated remediation docs

### PR DESCRIPTION
**Pull request by nhakmiller**
## Background

Jack pointed out that the automatic remediation docs were a bit outdated, so here is the new and improved version.

## Changes

- Updated the auto remediation docs with setup and usage instructions
- Added screenshots

## Testing

- N/A

### CODE REVIEW

File: docs/gitbook/policies/automatic-remediation/README.md [hunk 1]
```diff
@@ -4,21 +4,36 @@
...
 Panther supports Automatic Remediation of non-compliant resources to ensure that your infrastructure is as secure as possible. This works by:
 
-- Associating a remediation ID with a given Policy
+- Associating a remediation with a given Policy
 - When a Policy failure occurs, the `aws-remediation` Lambda assumes a role in the target account with the offending resource and performs the remediation
 
 The following diagram shows how Panther supports Automatic Remediation:
 
-![](../.gitbook/assets/autoremediationmulticustomeraccount.png)
+![](../../.gitbook/assets/autoremediationmulticustomeraccount.png)
```
Comment (austinbyers) [inline docs/gitbook/policies/automatic-remediation/README.md]: The text in brackets is the default text if the image can't be loaded or someone is using a screen reader, it's good practice to fill them in. For example,

```
![remediation-diagram](../../.gitbook/assets/...)
```

Here and for all images

File: docs/gitbook/policies/automatic-remediation/README.md [hunk 2]
```diff
@@ -4,21 +4,36 @@
...
 
 ## Setup
 
-{% hint style="info" %}
-Coming Soon!
-{% endhint %}
+Enabling automatic remediation for a Cloud Security source is simple. The only requirement is to check the `AWS Automatic Remediations` checkbox while on-boarding the Cloud Security source, and the pre-requisite role will be deployed as part of the on-boarding stack.
```
Comment (austinbyers) [inline docs/gitbook/policies/automatic-remediation/README.md]: `onboarding` and `prerequisite` are one word (no dash)

File: docs/gitbook/policies/automatic-remediation/README.md [hunk 3]
```diff
@@ -4,21 +4,36 @@
...
+
+Now, all future failures of the policy will automatically be re-mediated with the selected remediation. In order to apply the remediation to already detected failures, you can select the `Remediate` button on a failing resource when viewing the resources for the policy.
+
+![](../../.gitbook/assets/remediateButton.png)
+
+In order to apply the remediation to all currently failing resources, simply disable the policy then re-enable the policy to re-evaluate all resources immediately with the automatic remediation in place. Panther doesn't do this automatically for safety reasons. This way you are able to enable an automatic remediation, test it out on a few resources to make sure everything is working as intended, then apply it to all failing resources (if desired) with the confidence that the exact policy and remediation configurations you intend to carry out are working as intended.
```
Comment (austinbyers) [inline docs/gitbook/policies/automatic-remediation/README.md]: #FeatureNotABug :)

File: docs/gitbook/policies/automatic-remediation/README.md (hunk 2 above)
Comment (nhakmiller) [reply docs/gitbook/policies/automatic-remediation/README.md]: My IDE crabs at me when I make it one word :( I will just tell it to be quite.

**Pull request merged**